hat/conf/messages.pt

```diff
@@ -49,3 +49,15 @@ email.dataswift.auth.footer.financial.conduct.authority=e pela Financial Conduct
 email.dataswift.auth.reset.password.title=Redefinição de senha
 email.dataswift.auth.reset.password.follow.link=Para redefinir sua senha, siga este link:
 email.dataswift.auth.reset.password.link=Redefinir senha
+
+email.dataswift.auth.welcome.to.pda={0} recebe você em sua Personal Data Account
+email.dataswift.auth.pda.was.created=
+email.dataswift.auth.go.to.app=
+
+email.dataswift.auth.safe.place=Um local seguro para seus dados
+email.dataswift.auth.safely.legally.store=Sua Personal Data Account é onde você pode armazenar com segurança e possuir legalmente todos os dados.
+
+email.dataswift.auth.activate.text=Ative sua conta para continuar usando o {0} e descobrir outros aplicativos que você pode usar com seu PDA.
+email.dataswift.auth.start.here=
+
+email.dataswift.auth.issued.by=
```

Comment:
Do we not have these translated?  I think Product needs to engage a translation service.

Reply:
Even if we don't have it it should default to English shouldn't it?

Reply:
Updated the other values, only one I couldn't find is this `issued.by`

Reply:
Checking with Steve for it. I do remember a conversation for it if it gets translated or not.

Reply:
I found "Issued By": `Publicado por:`

I can just use this? The English has "Your PDA was issued by:"

Close enough? 😄

Reply:
Yup, you are good. Asked Steve.